backoffice/backoffice/workflows/api/views.py

```diff
@@ -127,6 +129,65 @@ def resolve(self, request, pk=None):
                 ResolutionDags[serializer.validated_data["value"]].label, pk, extra_data
             )
 
+    @action(detail=True, methods=["post"])
+    def restart(self, request, pk=None):
```

Comment:
this method is too long, can we split the functionality?

Reply:
split into function that are now implemented in `airflow_utils`